packages/twenty-server/src/workspace/messaging/message-channel-message-association/message-channel-message-association.service.ts

```diff
@@ -0,0 +1,116 @@
+import { Injectable } from '@nestjs/common';
+
+import { WorkspaceDataSourceService } from 'src/workspace/workspace-datasource/workspace-datasource.service';
+import { MessageChannelMessageAssociationObjectMetadata } from 'src/workspace/workspace-sync-metadata/standard-objects/message-channel-message-association.object-metadata';
+import { ObjectRecord } from 'src/workspace/workspace-sync-metadata/types/object-record';
+
+@Injectable()
+export class MessageChannelMessageAssociationService {
+  constructor(
+    private readonly workspaceDataSourceService: WorkspaceDataSourceService,
+  ) {}
+
+  public async getMessageChannelMessageAssociationsByMessageExternalIdsAndMessageChannelId(
```

Comment:
Those names are too long. This function will always be called using the `MessageChannelMessageAssociationService`. So I would rename for example this one `getByMessageExternalIdsAndMessageChannelId`.